### ISSUE TITLE
Usability: Top Bar Icons – add tooltips

### ISSUE THREAD

**Issue by jdittrich** [jmoenig/Snap--Build-Your-Own-Blocks#808]
The top bar icons may be hard to understand for new users. 

![Top Bar](https://cloud.githubusercontent.com/assets/3416487/7783824/420faea4-0150-11e5-8057-d2ea685e9e83.png)

A (probably easy-to-implement) way to help the user would be adding tooltips to these buttons.

I addition, it might be useful to consider that the icons influencing the size of the preview should probably be connected ("mapped") to the preview somehow (like being in a top-preview toolbar or semi-transparently floating in a corner of it). The position in the main toolbar suggest, that these icons influence the display of the whole IDE and not just a part (the preview).

![blocks](https://cloud.githubusercontent.com/assets/3416487/7783859/a0b2b568-0151-11e5-8b7f-37c64e99a058.png)

**Comment by jmoenig** [comment]
You're right, tooltips would be very easy to add, and I swear I had them at some point, but then took the out for some reason I've since forgotten :-)

I like you alternative suggestion, placing the controls  in more expressive locations. Thanks!
